docs/01_cva6_user/CVX_Interface_Coprocessor.rst

```diff
@@ -23,12 +23,249 @@ CV-X-IF Interface and Coprocessor
 
 CV-X-IF interface specification
 -------------------------------
-Need to step1 verification. TSS/Guillaume will do.
-Refer to the CV-X-IF specification, mention the 3 supported protocol interfaces, identify the CVA6 specific features.
+
+Description
+~~~~~~~~~~~
+This design specification presents global functionalities of
+Core-V-eXtension-Interface (XIF, CVXIF, X-interface) in the CVA6 core.
+
+.. code-block:: text
+
+   The CORE-V X-Interface is a RISC-V eXtension interface that provides a
+   generalized framework suitable to implement custom coprocessors and ISA
+   extensions for existing RISC-V processors.
+
+   --core-v-xif Readme, https://github.com/openhwgroup/core-v-xif
+
+The specification of the CVXIF bus protocol can be found at:
+https://docs.openhwgroup.org/projects/openhw-group-core-v-xif/en/core-v-xif_v0.2.0/x_ext.html

```

Comment:
Refer to [CV-X-IF] instead. We keep references up to date in the introduction file.